src/main/java/decodes/tsdb/algo/RelativeHumidity.java

```diff
@@ -0,0 +1,108 @@
+package decodes.tsdb.algo;
+
+import decodes.tsdb.DbCompException;
+import decodes.tsdb.IntervalCodes;
+import ilex.var.NamedVariable;
+import opendcs.opentsdb.Interval;
+import org.opendcs.annotations.algorithm.Algorithm;
+import org.opendcs.annotations.algorithm.Input;
+import org.opendcs.annotations.algorithm.Output;
+
+@Algorithm(
+		description ="Takes two inputs, Air temperature and dew point, to calculate relative humidity" )
```

Comment:
Description should just start right after the (, there doesn't need to be a new line here.